src/ensembl/src/shared/checkbox/Checkbox.tsx

```diff
@@ -58,7 +53,7 @@ const Checkbox = (props: Props) => {
         type="checkbox"
         className={defaultStyles.hiddenInput}
         onChange={handleOnChange}
-        defaultChecked={checkedStatus}
+        defaultChecked={props.checked}
```

Comment:
`defaultChecked={props.checked}` -> `value={props.checked}`

Reply:
I did try that before but I was getting the following error:

> Type 'boolean' is not assignable to type 'string | number | string[] | undefined'.

Reply:
`checked={props.checked}` works fine. I've updated it.

Reply:
Right. Makes sense.